Storage/SQL/SQLiteBookmarksModel.swift

```diff
@@ -855,4 +890,10 @@ public class MergedSQLiteBookmarks: BookmarksModelFactorySource {
         self.local = SQLiteBookmarks(db: db)
         self.buffer = SQLiteBookmarkBufferStorage(db: db)
     }
+
+    public func getURLForKeywordSearch(keyword: String) -> Deferred<Maybe<String>> {
+        return modelFactory >>== {
+            return ($0 as! KeywordSearchSource).getURLForKeywordSearch(keyword)
```

Comment:
If you really want to know:

https://gist.github.com/rnewman/82c17460161047db38d5